crates/re_viewer/src/app.rs

```diff
@@ -906,8 +906,6 @@ fn preview_files_being_dropped(egui_ctx: &egui::Context) {
 enum PanelSelection {
     #[default]
     Viewport,
-
-    EventLog,
```

Comment:
Same here: keeping the whole panel system in place even though we're back to one panel... for now